mallow_notifications/sns/schema/base.py

```diff
@@ -0,0 +1,29 @@
+from typing import Any, Dict, Optional, Union, get_origin
+
+from pydantic import BaseModel, Field
+
+from mallow_notifications.base.exceptions import NotificationError
+
+
+class BaseSchema(BaseModel):
+    def model_dump_without_none(self):
```

Comment:
use exclude None in side model_dump() instead of doing this this way, check Pydantic doc